shared/wallets/transaction/index.js

```diff
@@ -183,12 +183,20 @@ const AmountXLM = (props: AmountXLMProps) => {
   )
 }
 
-type TimestampProps = {|
+type TimestampLineProps = {|
+  error: ?string,
```

Comment:
lets treat these things as strings and just use '' for no error. same for statusDetail